products/BellHybrid/apps/application-bell-meditation-timer/presenter/SettingsPresenter.cpp

```diff
@@ -75,9 +76,14 @@ namespace app::meditation
                                                      utils::translate("app_bell_meditation_start_delay"),
                                                      utils::translate("common_second_lower")};
 
-        auto chimeVolume = new list_items::Numeric{list_items::Numeric::spinner_type::range{1, 10, 1},
-                                                   chimeVolumeModel,
-                                                   utils::translate("app_bell_meditation_chime_volume")};
+        constexpr auto volumeStep = 1U;
```

Comment:
Be careful when rebasing the PR, I've changed this area introducing a new range in my commit which was already merged into the master